templates/settings.tpl

```diff
@@ -37,6 +43,36 @@
 				{fbvElement type="checkbox" id="enabledCitationDownloads[]" value=$id checked=in_array($id, $enabledDownloads) label=$style translate=false}
 			{/foreach}
 		{/fbvFormSection}
+		{fbvFormArea title="plugins.generic.citationStyleLanguage.settings.citationUserGroups" class="pkpFormField--options"}
+			<p>{translate key="plugins.generic.citationStyleLanguage.settings.citationUserGroupsDescription"}</p>
+			{fbvFormSection list=true label="plugins.generic.citationStyleLanguage.settings.citationChooseAuthor"}
+				<p>{translate key='plugins.generic.citationStyleLanguage.settings.citationOptionChooseAuthor'}</p>
+				{foreach from=$allUserGroups item="group" key="id"}
+					{fbvElement type="checkbox" id="groupAuthor[]" value=$id checked=in_array($id, $groupAuthor) label=$group translate=false}
+				{/foreach}
+			{/fbvFormSection}
+			{if $application === 'omp'}
+				{fbvFormSection list=true label="plugins.generic.citationStyleLanguage.settings.citationChooseChapterAuthor"}
+					<p>{translate key='plugins.generic.citationStyleLanguage.settings.citationOptionChooseChapterAuthor'}</p>
+				{foreach from=$allUserGroups item="group" key="id"}
+					{fbvElement type="checkbox" id="groupChapterAuthor[]" value=$id checked=in_array($id, $groupChapterAuthor) label=$group translate=false}
+				{/foreach}
+				{/fbvFormSection}
+				{fbvFormSection list=true label="plugins.generic.citationStyleLanguage.settings.citationChooseEditor"}
+					<p>{translate key='plugins.generic.citationStyleLanguage.settings.citationOptionChooseEditor'}</p>
+					{foreach from=$allUserGroups item="group" key="id"}
+						{fbvElement type="checkbox" id="groupEditor[]" value=$id checked=in_array($id, $groupEditor) label=$group translate=false}
+					{/foreach}
+				{/fbvFormSection}
+			{/if}
+			{fbvFormSection list=true label="plugins.generic.citationStyleLanguage.settings.citationChooseTranslator"}
+				<p>{translate key='plugins.generic.citationStyleLanguage.settings.citationOptionChooseTranslator'}</p>
+				{foreach from=$allUserGroups item="group" key="id"}
+					{fbvElement type="checkbox" id="groupTranslator[]" value=$id checked=in_array($id, $groupTranslator) label=$group translate=false}
```

Comment:
This might eventually be something we need elsewhere in the application! But for now let's keep it here.